### DISCUSSION TITLE
Error When Opening Via Domain Name

### DISCUSSION THREAD

**Discussion by iamLazyCode** [louislam/dockge#300] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

on dockage when i click on the port number it should open the webpage right ?

![image](https://github.com/louislam/dockge/assets/13099434/3b9243e1-2e14-4bb4-8130-7ed72da57fe7)

but rather it uses dockage.xxxxx instead of set domain name

![image](https://github.com/louislam/dockge/assets/13099434/e07b4ae3-489b-42a7-8e97-910706b2f85d)

### 👟 Reproduction steps

Websites dont open

### 👀 Expected behavior

should open normally like

adc.domain-name.com

### 😓 Actual Behavior

should open normally like

adc.domain-name.com

### Dockge Version

1.3.3

### 💻 Operating System and Arch

ubuntu server 23.10

### 🌐 Browser

chromium 120.0.6099.71

### 🐋 Docker Version

Docker version 24.0.7, build afdd53b

**Comment by louislam** [comment]
There is no way to know the actual url.

You should add it manually under "URLs" inside "Edit".

**Comment by iamLazyCode** [comment]
> There is no way to know the actual url.
> 
> You should add it manually under "URLs" inside "Edit".

even after adding 

![image](https://github.com/louislam/dockge/assets/13099434/e5b5dd19-72c4-4ad3-a1c7-f087e12edc3d)

it opens with dockage.xxxxx

![image](https://github.com/louislam/dockge/assets/13099434/0cf39e26-748f-4c6e-83f6-67337f9760a2)

**Reply by louislam** [reply]
You should click this here:

![image](https://github.com/louislam/dockge/assets/1336778/903cea7b-7c34-4971-bda2-193ba6feda3b)

**Reply by iamLazyCode** [reply]
thankyou @louislam now it works :)

**Comment by iamLazyCode** [comment]
@louislam sir whenever i shut down my server and restart it i face this issue , i have to take down container and do `docker compose up -d` to make it work

![image](https://github.com/louislam/dockge/assets/13099434/7f4289e6-d35a-4cbc-b459-330b6ef6c5af)
